react/SummaryContext.tsx

```diff
@@ -1,7 +1,10 @@
 import React, { createContext, useContext, FunctionComponent } from 'react'
 
 interface ContextProps {
```

Comment:
This interface is almost identical to `SummaryProps`. Is it possible to use the same for both?

Reply:
done!